tests/integration/nm/ovs_test.py

```diff
@@ -378,6 +378,32 @@ def ovs_bridge_over_bond_system_iface_with_same_name(eth1_up, eth2_up):
     time.sleep(1)
 
 
+@pytest.fixture
+def ovs_bridge_and_ovs_interface_with_same_name_ipv4():
+    cmdlib.exec_cmd(
+        "nmcli c add type ovs-port conn.interface br-ex master br-ex con-name "
+        "ovs-port-br-ex".split(),
+        check=True,
+    )
+    cmdlib.exec_cmd(
+        "nmcli c add type ovs-bridge con-name br-ex conn.interface "
+        "br-ex".split(),
+        check=True,
+    )
+    nmcli_ovs_interface = (
+        "nmcli c add type ovs-interface slave-type ovs-port "
+        "conn.interface br-ex master ovs-port-br-ex con-name ovs-if-br-ex "
+        "ipv4.method manual ipv4.addr 192.0.2.2/24 ipv4.dns 192.0.2.1 "
+        "ipv4.routes".split()
+    )
+    nmcli_ovs_interface.append("0.0.0.0/0 192.0.2.1")
```

Comment:
using a tuple instead of a string with split() would make this simpler, then no append is needed. Why do you prefer the string+split?

Reply:
Well, in normal situations is easier to read IMO. But yes, here I will change it for a tuple.

Reply:
Done.